packages/hooks/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@jobber/hooks",
+  "version": "0.0.1",
+  "license": "MIT",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.js",
+  "scripts": {
+    "build": "tsc --project .",
+    "clean": "rm -rf dist/* tsconfig.tsbuildinfo",
+    "prepare": "npm run clean; npm run build"
+  },
+  "files": [
+    "dist/*"
+  ],
+  "devDependencies": {
+    "typescript": "^3.8.3"
+  },
+  "dependencies": {
+    "resize-observer-polyfill": "^1.5.1"
+  },
+  "peerDependencies": {
+    "react": "^16.8.6"
```

Comment:
will we need `@types/react` as a peer as well to handle our typescript?

Reply:
It seemed ok without it...